<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videodisplay</class>
 <widget class="QWidget" name="videodisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>videodisplay</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(231, 254, 255);</string>
  </property>
  <widget class="QLabel" name="displaytLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="waitBtn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>110</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>110</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_cat">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>190</y>
     <width>110</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>抓拍</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_broser">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>290</y>
     <width>110</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_return">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>380</y>
     <width>110</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
